packages/theming/src/utils/arrowStyles.ts

```diff
@@ -54,65 +36,77 @@ const animationStyles = (position: ArrowPosition, modifier: string) => {
 
 const positionStyles = (position: ArrowPosition, size: string, inset: string) => {
   const margin = math(`${size} / -2`);
-  const placement = math(`${margin} + ${inset}`);
+  const placement = math(`${margin} + ${inset} - 1`);
+  const _position = math(`${size} + 4`);
```

Comment:
Where does the 4 come from?

Perhaps renaming `_position` to `offset` could help better differentiate it from arrow `position`? 🤷‍♂️

Reply:
> Where does the 4 come from?

It was the unit needed to keep consistent with previous as it provided the means to bury a bigger arrow into the base element. TBH I'm not super happy about it.